cli/internal/commands/grant_permissions/generator.go

```diff
@@ -77,6 +80,7 @@ func (o *GeneratorOptions) addFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.CreateTrialNamespaces, "create-trial-namespace", o.CreateTrialNamespaces, "include trial namespace creation permissions")
 	cmd.Flags().StringVar(&o.NamespaceSelector, "ns-selector", o.NamespaceSelector, "bind to matching namespaces")
 	cmd.Flags().BoolVar(&o.IncludeManagerRole, "include-manager", o.IncludeManagerRole, "bind manager to matching namespaces")
+	cmd.Flags().BoolVar(&o.CreatePrometheusRBAC, "prometheus", true, "roles to create Prometheus in cluster")
```

Comment:
This should default to the current value in the struct, then to adjust the default when adding the subcommand ([here](https://github.com/thestormforge/optimize-controller/blob/4015d2b82c4acfa4c62a1e7e46df3239cbfc4dd8/cli/internal/commands/generate/generate.go#L44) and [here](https://github.com/thestormforge/optimize-controller/blob/4015d2b82c4acfa4c62a1e7e46df3239cbfc4dd8/cli/internal/commands/initialize/generator.go#L179])).